### ISSUE TITLE
PTerClub

### ISSUE THREAD

**Issue by RoloSoze** [Jackett/Jackett#7950]
**Indexer Name**
PTerClub

**Indexer URL**
https://pterclub.com/

**Interface Language**
English, Chinese (Simplified, Traditional) 

**Type**
YAML (NexusPHP)

**Access**
Private - Invite sent to @garfield69 

**Notes**
- Supports IMDb ID Search
- Displays IMDb URL in the search results
- Please ensure the inclusion of dead torrents for all categories

**Comment by RoloSoze** [comment]
@garfield69 Invite sent!

**Comment by garfield69** [comment]
Registration and login successful. Thank you.

**Comment by garfield69** [comment]
Jackett 0.14.376

**Comment by RoloSoze** [comment]
@garfield69 I'm relaying a question from the staff about the categories:

They were wondering whether 'MV' or 'Music Video' can be listed as a category in Jackett for PTerClub in that exact name. As in, in the 'Add indexer' page where a list of categories is shown next to each Indexer, they would prefer to have Music Video added to the list of PTerClub Categories: "Music Video, Console, Movies, Audio, PC, TV, Other, Books"

They said that the main aim of PTer is to actually be a Music Video tracker, so the 'MV' category is very important for them despite the fact that they already have a lot of torrents in other categories. 

I said I'll ask about this, but I also mentioned that I don't think this is possible since MV is not a category defined in Jackett, and that every torrent under that category is mapped/linked to the closest/most similar category.

Let me know, please. Thanks.

**Comment by garfield69** [comment]
TLDR;
Unfortunately there is nothing can be done about this
DETAILED;
you are talking about 
![image](https://user-images.githubusercontent.com/6933240/78172171-15fb1a80-74b2-11ea-9b15-94734169b6a7.png)

Jackett implements the [Torznab](https://github.com/Sonarr/Sonarr/wiki/Implementing-a-Torznab-indexer) (with [nZEDb](https://github.com/nZEDb/nZEDb/blob/dev/docs/newznab_api_specification.txt) category numbering) API and has to abide by these specifications. 
The Torznab categories do not include one labelled `Music Video`, so when mapping a Web Site's category to Torznab we try to match the closest similar category, which in this case is `Audio/Video (3020)`
https://github.com/nZEDb/nZEDb/blob/c18d440e5cdc4aa11d48d5f2d74af147957001fc/docs/newznab_api_specification.txt#L658
On the add-indexer page, Jackett only shows the parent category of the indexer, as there is not enough room to list every category (some sites like the Russian one with Forum categories have hundreds!), and so the subcategory _Audio/Video_ gets reduced to just the parent category _Audio_.

But I can change the PTerClub Indexer description (the tooltip in the first picture above) and the header in the Config below
https://github.com/nZEDb/nZEDb/blob/c18d440e5cdc4aa11d48d5f2d74af147957001fc/docs/newznab_api_specification.txt#L658
to be
`PTerClub is a CHINESE Private Torrent Tracker for MUSIC VIDEO` if you prefer.

**Comment by RoloSoze** [comment]
> But I can change the PTerClub Indexer description (the tooltip in the first picture above) and the header in the Config below to be 
`PTerClub is a CHINESE Private Torrent Tracker for MUSIC VIDEO` if you prefer.

Yeah, I think this would work, but let me get back to you with the exact description that they prefer. 'Music Videos' would have to be there - since they primarily would like to be known for that - but it's not going to be the only item listed. I'll get back to you on this.

**Comment by RoloSoze** [comment]
@garfield69 I'm still waiting for the staff to get back to me about this. Meanwhile, please make sure you download at least one torrent, no matter how small, to avoid losing your account. You only need to do this once. Thanks.

**Comment by RoloSoze** [comment]
@garfield69 I never heard back from the staff, so let's just say: 
`PTerClub is a CHINESE Private Torrent Tracker for HD MUSIC VIDEOS, MOVIES, TV & ANIME`

**Comment by garfield69** [comment]
https://pterclub.com/login.php returning 
![image](https://user-images.githubusercontent.com/6933240/79110385-69346d80-7dce-11ea-9e96-9ab6a86a4b8b.png)

**Comment by RoloSoze** [comment]
Back online!

**Comment by garfield69** [comment]
I am still seeing 
![image](https://user-images.githubusercontent.com/6933240/79155284-fe625100-7e24-11ea-863e-309b35d0e397.png)
can you confirm you can actually login in please?
and if you can, double check the address?

**Comment by RoloSoze** [comment]
I'm able to log in, browse, download, etc. using the same URL: `https://pterclub.com/`
So yeah I can confirm that everything is normal. However, the homepage has does say the following:

![image](https://user-images.githubusercontent.com/23747425/79155674-7d14ab00-7d9f-11ea-9527-896537c0de2c.png)

So I guess if you logged out recently, you won't be able to log back in for a while until whatever issues they're having are resolved.

**Comment by garfield69** [comment]
Ah. I see. thank you.

**Comment by garfield69** [comment]
it does look like they have a disconnect between the default login path https://pterclub.com/login.php and the server which from the error meaage is looking for /login.asp

**Comment by RoloSoze** [comment]
> https://pterclub.com/login.php

This link is sending me to the same HTTP 404 page that you're seeing.

**Comment by garfield69** [comment]
changed my mind. 
it does not matter if the site is still up, until people can login this site is unavailable.

**Comment by RoloSoze** [comment]
No problem. Let's just make sure the indexer support doesn't get deleted in a month if this issue continues that long.

**Comment by RoloSoze** [comment]
@garfield69 I noticed that variations of this login issue are happening with multiple Chinese trackers, so I asked. I was told this is not an issue. April 26 is World Intellectual Property Day, so login prevention from early April to early May is a measure taken by some Chinese trackers to protect themselves. Things will go back to normal afterward.

**Comment by RoloSoze** [comment]
Update:
![image](https://user-images.githubusercontent.com/23747425/80623234-e2ab9d00-8a17-11ea-9f6e-962ca16b820a.png)

**Comment by RoloSoze** [comment]
> 
> 
> it does look like they have a disconnect between the default login path https://pterclub.com/login.php and the server which from the error meaage is looking for /login.asp

It's working now. I'll close this. If it's not working for you @garfield69, feel free to reopen it.

**Comment by RoloSoze** [comment]
@garfield69 Just realized the Power User Class in this tracker is named after you 🤣 

![image](https://user-images.githubusercontent.com/23747425/81444439-23f12a80-9145-11ea-9a9d-9aa42dbda66e.png)

**Comment by garfield69** [comment]
https://pterclub.com/ returning Error 504 Ray Gateway time-out

**Comment by ilike2burnthing** [comment]
back up

**Comment by Toguska** [comment]
@garfield69 please send me an invite to the PTerClub.

**Comment by garfield69** [comment]
@Toguska  This is not a forum for soliciting invitations, its a support site for the Jackett app. Please look elsewhere.
Besides, I don't have invites to give away.

**Comment by garfield69** [comment]
- https://pterclub.com/ SSL handshake failed Error code 525 
-  Web server is down Error code 521

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
pterclub is doing the same thing as turktorrent #4268, which is that they both appear to be forcing a cloudflare check in front of the torrents page, and since both indexer require the cookies method for access, despite passing the CF challenge the cookie fetched from the browser is invalidated.

**Comment by garfield69** [comment]
https://pterclub.com/ Site is down for maintenance, please check back again later... thanks

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://pterclub.com/ 
```
临时维护
系统临时维护中, 请稍后再[重载](javascript:window.location.reload();)此页面 -_-

PterClub is under temporary maintenance, please wait a few seconds and [refresh](javascript:window.location.reload();) this page later.
```

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
- https://pterclub.com/ cloudflare Origin is unreachable Error code 523
- uptime reports 403 forbidden across all checkpoints

**Comment by garfield69** [comment]
https://pterclub.com/ > https://pterclub.com/lander (Domain Expired)

**Comment by garfield69** [comment]
can anyone get https://pterclub.net/ to login? I get bounced back to the login page with no error or acknowledgment the login took place.

**Comment by christaikobo** [comment]
Yes I did. The website works.

**Comment by garfield69** [comment]
Thanks. Working for me now too.
Looks like the com tld is expired, I will remove that sitelink from the indexer.

**Comment by GaTuchoHD** [comment]
Another tracker has gone into silent mode; I have access to the tracker.

This was stated on the website home page:

We close the login page and invite system, please use the tg-bot if you need.

**Comment by garfield69** [comment]
the pterclub indexer already uses the cookie method.
